chapters/classes.tex

```diff
@@ -729,7 +729,7 @@ \subsection{Short Class Definitions}\label{short-class-definitions}
 \end{lstlisting}
 \end{example}
 
-\subsection{Restriction on combining base classes and other elements}\label{restriction-on-combining-base-classes-and-other-elements}
+\subsection{Combining base classes and other elements}\label{restriction-on-combining-base-classes-and-other-elements}\label{combining-base-classes-and-other-elements}
```

Comment:
Capitalization:
```suggestion
\subsection{Combining Base Classes and Other Elements}\label{restriction-on-combining-base-classes-and-other-elements}\label{combining-base-classes-and-other-elements}
```
There is also uncapitalized "Unspecified enumeration" on line 1653 of this file/ But GutHub does not allow me to comment there because it is not part of this change.